tests/pipelines/test_split_step.py

```diff
@@ -128,6 +128,7 @@ def test_from_pipeline_config_works_with_target_col(tmp_path):
         assert SplitStep.from_pipeline_config({"target_col": "fake_col"}, "fake_root") is not None
 
 
+@pytest.mark.mlp_next_release
```

Comment:
Just testing out the decorator. This will be used to indicate that a test should only be run against the next MLP release branch. These decorators should then be removed when it comes time to make the next MLP release.

TODO before merging: Add a version specifier so that these tests will always run if the MLflow dev version is >= than the specified test next release version

ight